### PR TITLE
Fix song select tests not waiting for beatmap imports to arrive

### PR DESCRIPTION
**Pull request by peppy**
After the change to realm, notification fires could take a frame or two.
We aren't accounting for this.

Fixes test failures like https://github.com/ppy/osu/runs/4963255990?check_suite_focus=true

### CODE REVIEW

File: osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs [hunk 1]
```diff
@@ -73,6 +74,17 @@ public override void SetUpSteps()
...
 
+        public override void TearDownSteps()
+        {
+            base.TearDownSteps();
+
+            AddStep("remove song select", () =>
+            {
+                songSelect?.Expire();
+                songSelect = null;
+            });
```
Comment (smoogipoo) [inline osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs:85]: Why is this required/why doesn't `base.TearDownSteps()` forcefully exiting screens in the stack handle this?

Comment (peppy) [reply osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs:85]: Ah, didn't realise this was a screen test.

The important part is the `null` set, so the next run doesn't see the previous run's (expired) `songSelect` and try to match against it (see the special case of the newly adding condition, where we aren't always guaranteed the song select has been created yet).

Can reduce this just to the null set if that reads better.

Comment (smoogipoo) [reply osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs:85]: Maybe best to do that null set in `Setup` or `SetupSteps`?

Comment (peppy) [reply osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs:85]: Sure, will move it. Had that in a previous iteration.

**Review by smoogipoo — APPROVED** [review]
